app/__init__.py

```diff
@@ -1,17 +1,18 @@
 """ app/__init__.py """
 
 from flask import Flask
-from app import useraccounts,shoppinglist, shoppingitems
+from app import useraccounts, shoppinglist, shoppingitems
 
 # Initialize the app
-app = Flask(__name__, instance_relative_config=True)
+app = Flask(  # pylint: disable=invalid-name
+    __name__, instance_relative_config=True)
 app.secret_key = 'dresscodesleepbehappy'
 
-user_object = useraccounts.UserClass()
-shoplist_obj = shoppinglist.ShoppinglistClass()
-shopitems_obj = shoppingitems.ShoppingItemsClass()
+user_object = useraccounts.UserClass()  # pylint: disable=invalid-name
```

Comment:
instead of disabling this pylint error can you use pylintrc file to set your desired `good-names` variables